ush/jediinc2fv3.py

```diff
@@ -57,22 +60,136 @@ def jedi_inc_to_fv3(FV3JEDIinc, FV3inc):
             for name, variable in ncin.variables.items():
                 if len(variable.dimensions) == 4:
                     dimsout = variable.dimensions[1:]
+                    dimsout4 = dimsout
+                elif len(variable.dimensions) == 3:
+                    dimsout = variable.dimensions[1:]
+                    dimsout3 = dimsout
                 else:
                     dimsout = variable.dimensions
+
                 if name in vardict:
+                    if name == 'delp':
```

Comment:
code tidy here maybe? 
```python
if name in ['delp','DELP','delz']:
  continue
```
Or better yet, do your variable name mapping with a dictionary?

Reply:
Good suggestion @guillaumevernieres .    After examining the script, I see that delp and delz can be removed from the referenced section and from vardict.   These changes were committed at [dbd3580](https://github.com/NOAA-EMC/GDASApp/pull/139/commits/dbd35805cb9c3cd88d3156659f68a0bc61b189e5).